prometheus.lua

```diff
@@ -76,6 +76,66 @@ local DEFAULT_BUCKETS = {0.005, 0.01, 0.02, 0.03, 0.05, 0.075, 0.1, 0.2, 0.3,
 -- Prefix for internal shared dictionary items.
 local KEY_INDEX_PREFIX = "__ngx_prom__"
 
+-- Accept range for second byte of utf8
+local accept_range = {
```

Comment:
Hello, leave the `accept_range` table outside `validate_utf8_string` as a const variable is mainly for performance consideration, so that we can avoid creating and destroying frequently. 
will add a comment later.

Reply:
Cool, thanks for clarifying. Could you add a comment explaining this please?

Reply:
sure